service.py

```diff
@@ -81,6 +81,8 @@ def transform_event_to_docs(event, index, omit):
                     "searchableInterests": [
                         {"name": p} for p in (payload.get("interests") or [])
                     ],
+                    "linkedin": payload.get("linkedin", ""),
```

Comment:
This has nothing to do with the deployment configuration but I wanted to kill 2 birds with 1 stone and add this small fix.

Reply:
In my mind, this should be a separate PR, or at least a separate commit, but it's a really small change.